css/src/atomics/colors.scss

```diff
@@ -0,0 +1,213 @@
+// Framework colors
+
+@each $name, $color-set in $colors {
+	$base: nth($color-set, $effect-index-base);
+	$background: nth($color-set, $effect-index-background);
+	$dark: nth($color-set, $effect-index-dark);
+	$hover: nth($color-set, $effect-index-hover);
+	$active: nth($color-set, $effect-index-active);
+	$invert: nth($color-set, $effect-index-invert);
+
+	.text-color-#{$name} {
+		color: $base !important;
+	}
+
+	a.text-color-#{$name} {
+		&:hover,
+		&.focus-visible {
+			color: $hover !important;
+		}
+
+		&:active {
+			color: $active !important;
+		}
+	}
+
+	.text-color-#{$name}-invert {
+		color: $invert !important;
+	}
+
+	.background-color-#{$name} {
+		outline-color: $invert;
+		background-color: $base !important;
+	}
+
+	.background-color-#{$name}-tablet {
+		@include tablet {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-desktop {
+		@include desktop {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-widescreen {
+		@include widescreen {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-invert {
+		outline-color: $base;
+		background-color: $invert !important;
+	}
+
+	.background-color-#{$name}-light {
+		outline-color: $dark;
+		background-color: $background !important;
+	}
+
+	.background-color-#{$name}-dark {
+		outline-color: $background;
+		background-color: $dark !important;
+	}
+}
+
+// Text color
+
+.text-color {
```

Comment:
- `.text-subtle, .text-overlay-invert` comes from "helpers > typography.scss" in docs-ui. They were not moved to the typography atomics in Atlas and they're color related so I added them here.